### PR TITLE
python3Packages.sphinxcontrib-mscgen: init at 0.6

### PR DESCRIPTION
**Pull request by drupol**
Hi,

This is my first Python derivation.

URL of the project: https://github.com/sphinx-contrib/mscgen/

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/sphinxcontrib-mscgen/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  doCheck = false;
+
+  pythonImportsCheck = [
+    "sphinxcontrib.mscgen"
+  ];
+
+  meta = with lib; {
+    description = "Sphinx extension using mscgen to render diagrams";
+    homepage = "https://github.com/sphinx-contrib/mscgen";
+    license = licenses.publicDomain;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/sphinxcontrib-mscgen/default.nix:36]: missing maintainer

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by mweinelt** [comment]
> pythonPackages.sphinxcontrib-mscgen: init at 0.6 

`pythonPackages` references `python2.pkgs` basically.

File: pkgs/development/python-modules/sphinxcontrib-mscgen/default.nix (hunk 1 above)
Comment (drupol) [reply pkgs/development/python-modules/sphinxcontrib-mscgen/default.nix:36]: Just added !

**Comment by drupol** [comment]
Tested successfully locally.

File: pkgs/development/python-modules/sphinxcontrib-mscgen/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
```
Comment (mweinelt) [inline pkgs/development/python-modules/sphinxcontrib-mscgen/default.nix:36]: Wondering if we should introduce `BOLA` as a license, so others don't have to look it up.

Comment (drupol) [reply pkgs/development/python-modules/sphinxcontrib-mscgen/default.nix]: Done.

**Comment by drupol** [comment]
Thanks!

**Pull request merged**